src/mesh/interpolation/hermite_spline_xz.cxx

```diff
@@ -346,6 +358,8 @@ Field3D XZHermiteSpline::interpolate(const Field3D& f, const std::string& region
   ASSERT1(f.getMesh() == localmesh);
   Field3D f_interp{emptyFrom(f)};
 
+  // TODO(peter): Should we apply dirichlet BCs to derivatives?
+
```

Comment:
I do not think so. What would be the motivation? So we do not need two x-guard cells?